.github/ISSUE_TEMPLATE/election.md

```diff
@@ -0,0 +1,128 @@
+
+### 📝 Nominator Checklist
+
+- [ ] Has communicated with the nominee and received their consent
+- [ ] Information provided is complete and accurate
+- [ ] Clear reasons for nomination and contribution history provided
+- [ ] Confirmed that the nominee has time and willingness to take on the responsibilities
+- [ ] Has informed the nominee about the election process and requirements
+
+### ℹ️ Additional Information
+
+- Election process and role descriptions can be found in the relevant repository documentation
+- Please provide additional comments in this issue for any objections or special circumstances
+- During the voting period, all Maintainers and Committers have voting rights
+- Election results require a **simple majority** (over 50%) to pass
+- The voting process is open and transparent, and results will be announced to the community
+
+## 🏆 Candidate Profile
+
+### 📊 Current Role and Contributions
+
+Please briefly describe the nominee's role in the community, projects they've participated in, and their main contributions.
+
+<!-- Please include the following information:
+- History of participation in the Jeandle community
+- Main projects and modules contributed to
+- Code contribution statistics (number of PRs, lines of code, etc.)
+- Participation in issue discussions and assistance provided
+- Community activity participation
+-->
+
+### 💡 Reasons for Nomination
+
+Please explain why you recommend this candidate and why you believe they are suitable for this role.
+
+<!-- Please consider the following aspects:
+- Technical skills and expertise
+- Community responsibility and engagement
+- Communication and collaboration abilities
+- Leadership and influence
+- Long-term commitment to the community
+-->
+
+### 🚀 Candidate's Future Plans
+
+If elected, please briefly describe the candidate's plans and goals for the community.
+
+<!-- May include:
+- Technical areas they plan to focus on
+- Community projects or improvements they wish to drive
+- Plans for mentoring new contributors
+- Ideas for community building and promotion
+- Expected time and effort commitment
+-->
+
+### 🎖️ Role Application
+
+<!-- Please select one: -->
+- [ ] Committer
+- [ ] Maintainer
+- [ ] Other Role: ______
+
+#### Role Descriptions
+
+- **Committer**: Has repository write access, can merge PRs, participates in daily maintenance
+- **Maintainer**: Has higher privileges, responsible for project direction, release management, community governance
+
+## 📊 Voting Results
+
+| Voter  | Vote      | Notes         |
+| ------ | --------- | ------------- |
+| @user1 | ✅ Approve | Notes go here |
+| @user2 | ✅ Approve | Notes go here |
+| @user3 | ❌ Reject  | Notes go here |
+
+---
+
+**Voting Instructions:**
+
+- ✅ Approve: Support the candidate for this role
+- ❌ Reject: Do not support the candidate for this role
+- 🤔 Abstain: Do not participate in this vote
+
+_Last updated: 2025-08-29_
```

Comment:
This line won't appear in the template when creating such a issue, right?

Reply:
After we apply the automated tool （such as https://github.com/cncf/gitvote）, this component no longer needs to be maintained manually, so I will remove it.